pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -1,3 +1,71 @@
 package preflight
 
+import (
+	"errors"
+	"fmt"
+	crcos "github.com/code-ready/crc/pkg/os"
```

Comment:
at the end of the import list please

Reply:
This is `gofmt` which put it on this place :(